Sources/RoktUXHelper/Data/Model/Constants.swift

```diff
@@ -86,3 +86,13 @@ let kProgressIndicatorAnnouncement = "%d of %d"
 let kCloseButtonAnnouncement = "Close button"
 let kNextPageButtonAnnouncement = "Next page button"
 let kPreviousPageButtonAnnouncement = "Previous page button"
+
+// MARK: Cart Item Instant purchase constants
+let kCartItemId = "cartItemId"
```

Comment:
Can these be encapsulated into a struct or similar as they are tightly tied to just one thing. This would stop the creep of growing constants

Reply:
Ace, thank you!